Index: src/main/java/cn/zhku/waterfowl/modules/material/controller/MaterialController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/cn/zhku/waterfowl/modules/material/controller/MaterialController.java	(revision c7224ea6e4a39373f0879b0ca2061c944149c458)
+++ src/main/java/cn/zhku/waterfowl/modules/material/controller/MaterialController.java	(date 1512373770880)
@@ -21,6 +21,7 @@
 
 import javax.servlet.http.HttpServletRequest;
 import java.io.File;
+import java.sql.Timestamp;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
@@ -47,6 +48,8 @@
     @RequestMapping("add")
     public Message addMaterial(Material material) throws Exception {
         material.setIdStorage(UUID.randomUUID().toString().replace("-", "").toUpperCase());   //用32位长度的UUID来设置用户id
+        Timestamp t = new Timestamp(System.currentTimeMillis());
+        material.setDate(t);
         if (materialService.add(material) == 1)
             return new Message("1", "插入材料成功");
         else
@@ -118,7 +121,7 @@
      * @throws Exception sql
      */
     @ResponseBody
-    @RequestMapping("selectBy")
+    @RequestMapping("list")
     public PageInfo<Material> select(Material material, CommonQo commonQo) throws Exception {
         //  设置页码，页面大小，排序方式,此处的sql相当于 limit pageNum ,pageSize orderBy id desc
         PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(), "date desc");
@@ -133,38 +136,43 @@
     @RequestMapping("show")
     public PageInfo<Material> show(Material entity, CommonQo commonQo) throws Exception {
 //        //  设置默认的排序，如果前端需要排训查询，则加上参数  sort = 数据库字段 ，
-//        if (commonQo.getSort().equals("1"))  //字符串"1"是sort的默认值，相当于 order by 1 ,即按主键排序
-//        commonQo.setSort("date desc");
+        if (commonQo.getSort().equals("1"))  //字符串"1"是sort的默认值，相当于 order by 1 ,即按主键排序
+        {
+            commonQo.setSort("date desc");
+        }
+        else if(commonQo.getSort().equals("2")){
+            commonQo.setSort("expiration_date desc");
+        }
 //        PageHelper.startPage(commonQo.getPageNum(),commonQo.getPageSize(),commonQo.getSort());
 //
 //        return new PageInfo<>(materialService.list(entity,commonQo));
 //    }
         //  设置页码，页面大小，排序方式,此处的sql相当于 limit pageNum ,pageSize orderBy id desc
-        PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(), "date desc");
+        PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(), commonQo.getSort());
         //  通过服务层获取查询后的用户列表
         List<Material> materialList = materialService.showAll(entity, commonQo);
         //  返回 pageBean实体
         return new PageInfo<Material>(materialList);
     }
 
-    @ResponseBody
-    @RequestMapping("list")
-    public PageInfo<Material> list(Material entity, CommonQo commonQo) throws Exception {
-//        //  设置默认的排序，如果前端需要排训查询，则加上参数  sort = 数据库字段 ，
-//        if (commonQo.getSort().equals("1"))  //字符串"1"是sort的默认值，相当于 order by 1 ,即按主键排序
-//        commonQo.setSort("date desc");
-//        PageHelper.startPage(commonQo.getPageNum(),commonQo.getPageSize(),commonQo.getSort());
-//
-//        return new PageInfo<>(materialService.list(entity,commonQo));
+//    @ResponseBody
+//    @RequestMapping("list")
+//    public PageInfo<Material> list(Material entity, CommonQo commonQo) throws Exception {
+////        //  设置默认的排序，如果前端需要排训查询，则加上参数  sort = 数据库字段 ，
+////        if (commonQo.getSort().equals("1"))  //字符串"1"是sort的默认值，相当于 order by 1 ,即按主键排序
+////        commonQo.setSort("date desc");
+////        PageHelper.startPage(commonQo.getPageNum(),commonQo.getPageSize(),commonQo.getSort());
+////
+////        return new PageInfo<>(materialService.list(entity,commonQo));
+////    }
+//        //  设置页码，页面大小，排序方式,此处的sql相当于 limit pageNum ,pageSize orderBy id desc
+//        PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(), "expiration_date desc");
+//        //  通过服务层获取查询后的用户列表
+//        List<Material> materialList = materialService.listAll(entity, commonQo);
+//        //  返回 pageBean实体
+//        return new PageInfo<Material>(materialList);
 //    }
-        //  设置页码，页面大小，排序方式,此处的sql相当于 limit pageNum ,pageSize orderBy id desc
-        PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(), "expiration_date desc");
-        //  通过服务层获取查询后的用户列表
-        List<Material> materialList = materialService.listAll(entity, commonQo);
-        //  返回 pageBean实体
-        return new PageInfo<Material>(materialList);
-    }
-    // 接受一个新Excel
+//    // 接受一个新Excel
 
     /**
      * 导入用excel
@@ -227,7 +235,7 @@
         ExportExcelUtil<Material> exportExcelUtil = new ExportExcelUtil<>();
 
         List<Material> materialList = materialService.findList(material);
-        String[] headers = {"物资编号", "进厂时间", "物资名称", "物资有效期", "关联厂商", "联系电话", "是否过期", "物资数量", "计数单位", "存储地点", "存储方式", "备注", "记录者编号", "登录者编号"};
+        String[] headers = {"物资编号", "进厂时间", "物资名称", "物资有效期", "关联厂商", "联系电话", "是否过期", "物资数量", "计数单位", "存储地点", "存储方式", "备注", "登录者编号", "负责人编号"};
         //  通过标题和数据库数据生成XLS文件
         //Workbook wb = exportExcelUtil.exportXLS("用户表单",headers,userList);
         // 直接调用工具类生成xls或xlsx文件,用户访问此链接直接下载
Index: src/main/java/cn/zhku/waterfowl/modules/material/model/MaterialExcel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/cn/zhku/waterfowl/modules/material/model/MaterialExcel.java	(revision c7224ea6e4a39373f0879b0ca2061c944149c458)
+++ src/main/java/cn/zhku/waterfowl/modules/material/model/MaterialExcel.java	(date 1512370684323)
@@ -42,10 +42,10 @@
     @EntiName(RName = "备注")
     private String remark;
 
-    @EntiName(RName = "记录者编号")
-    private String id_recorder;
-
     @EntiName(RName = "登录者编号")
+    private String id_recorder;
+
+    @EntiName(RName = "负责人编号")
     private String id_charge;
 
 
Index: src/main/java/cn/zhku/waterfowl/modules/material/service/MaterialService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/cn/zhku/waterfowl/modules/material/service/MaterialService.java	(revision c7224ea6e4a39373f0879b0ca2061c944149c458)
+++ src/main/java/cn/zhku/waterfowl/modules/material/service/MaterialService.java	(date 1512370099472)
@@ -145,15 +145,16 @@
             criteria.andDateLessThanOrEqualTo(commonQo.getEnd());
         return materialMapper.selectByExample(materialExample);
     }
-    public List<Material> listAll(Material entity, CommonQo commonQo) {
-        MaterialExample materialExample = new MaterialExample();
-        MaterialExample.Criteria criteria = materialExample.createCriteria();
-        //  根据入厂时间区间来查找
-        if (commonQo.getStart() != null)
-            criteria.andExpirationDateGreaterThanOrEqualTo(commonQo.getStart());
-        if (commonQo.getEnd() != null)
-            criteria.andExpirationDateLessThanOrEqualTo(commonQo.getEnd());
-        return materialMapper.selectByExample(materialExample);
-    }
+//    public List<Material> listAll(Material entity, CommonQo commonQo) {
+//        MaterialExample materialExample = new MaterialExample();
+//        MaterialExample.Criteria criteria = materialExample.createCriteria();
+//        //  根据入厂时间区间来查找
+//        if (commonQo.getStart() != null)
+//            criteria.andExpirationDateGreaterThanOrEqualTo(commonQo.getStart());
+//        if (commonQo.getEnd() != null)
+//            criteria.andExpirationDateLessThanOrEqualTo(commonQo.getEnd());
+//        return materialMapper.selectByExample(materialExample);
+//    }
+
 
 }
Index: src/main/java/cn/zhku/waterfowl/modules/outStorage/controller/OutStorageController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/cn/zhku/waterfowl/modules/outStorage/controller/OutStorageController.java	(revision c7224ea6e4a39373f0879b0ca2061c944149c458)
+++ src/main/java/cn/zhku/waterfowl/modules/outStorage/controller/OutStorageController.java	(date 1512373840689)
@@ -20,6 +20,7 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
 
+import java.sql.Timestamp;
 import java.util.List;
 import java.util.UUID;
 
@@ -43,6 +44,8 @@
         public Message addOutstorage(Outstorage outstorage) throws Exception {
             outstorage.setIdOutstorage(
                     UUID.randomUUID().toString().replace("-","").toUpperCase());   //用32位大小的UUID来设置用户id
+            Timestamp t = new Timestamp(System.currentTimeMillis());
+            outstorage.setRecordDate(t);
             if(outStorageService.add(outstorage) == 1)
                 return new Message("1","添加出库记录成功");
             else
@@ -100,7 +103,7 @@
          * @throws Exception    sql
          */
         @ResponseBody
-        @RequestMapping("selectBy")
+        @RequestMapping("list")
         public PageInfo<Outstorage> select(Outstorage outstorage, CommonQo commonQo) throws Exception {
             //  设置页码，页面大小，排序方式,此处的sql相当于 limit pageNum ,pageSize orderBy id desc
             PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(),"record_date desc");
Index: src/main/java/cn/zhku/waterfowl/modules/poultry/controller/PoultryController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/cn/zhku/waterfowl/modules/poultry/controller/PoultryController.java	(revision c7224ea6e4a39373f0879b0ca2061c944149c458)
+++ src/main/java/cn/zhku/waterfowl/modules/poultry/controller/PoultryController.java	(date 1512373889944)
@@ -12,6 +12,7 @@
 import com.github.pagehelper.PageHelper;
 import com.github.pagehelper.PageInfo;
 import com.sun.tools.doclets.formats.html.PackageFrameWriter;
+import org.apache.poi.ss.formula.functions.T;
 import org.springframework.beans.BeanUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
@@ -23,6 +24,7 @@
 
 import javax.servlet.http.HttpServletRequest;
 import java.io.File;
+import java.sql.Timestamp;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
@@ -44,6 +46,8 @@
         @RequestMapping("add")
         public Message addPoultry(Poultry poultry) throws Exception {
             poultry.setIdPoultry(UUID.randomUUID().toString().replace("-", "").toUpperCase());   //用32位长度的UUID来设置用户id
+            Timestamp t = new Timestamp(System.currentTimeMillis());
+            poultry.setRecordDate(t);
             if (poultryService.add(poultry) == 1)
                 return new Message("1", "插入材料成功");
             else
@@ -115,7 +119,7 @@
          * @throws Exception sql
          */
         @ResponseBody
-        @RequestMapping("selectBy")
+        @RequestMapping("list")
         public PageInfo<Poultry> select(Poultry poultry, CommonQo commonQo) throws Exception {
             //  设置页码，页面大小，排序方式,此处的sql相当于 limit pageNum ,pageSize orderBy id desc
             PageHelper.startPage(commonQo.getPageNum(), commonQo.getPageSize(), "record_date desc");
@@ -205,7 +209,7 @@
         ExportExcelUtil<Poultry> exportExcelUtil = new ExportExcelUtil<>();
 
         List<Poultry> poultryList = poultryService.findList(poultry);
-        String[] headers = {"入库编号", "记录时间", "家禽类型", "数量", "单位", "关联厂商", "联系电话", "备注", "记录者编号", "登录者编号"};
+        String[] headers = {"入库编号", "记录时间", "家禽类型", "数量", "单位", "关联厂商", "联系电话", "备注", "登录者编号", "负责人编号"};
         //  通过标题和数据库数据生成XLS文件
         //Workbook wb = exportExcelUtil.exportXLS("用户表单",headers,userList);
         // 直接调用工具类生成xls或xlsx文件,用户访问此链接直接下载
Index: src/main/java/cn/zhku/waterfowl/modules/poultry/model/PoultryExcel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/cn/zhku/waterfowl/modules/poultry/model/PoultryExcel.java	(revision c7224ea6e4a39373f0879b0ca2061c944149c458)
+++ src/main/java/cn/zhku/waterfowl/modules/poultry/model/PoultryExcel.java	(date 1512370482249)
@@ -34,10 +34,10 @@
     @EntiName(RName = "备注")
     private String remark;
 
-    @EntiName(RName = "记录者编号")
-    private String id_recorder;
-
     @EntiName(RName = "登录者编号")
+    private String id_recorder;
+
+    @EntiName(RName = "负责人编号")
     private String id_charge;
 
     public Integer getId_poultry() {
